### PR TITLE
Introducing `attractor calc --format=csv`, which exports data as CSV

### PR DESCRIPTION
**Pull request by jodosha**
Hey @julianrubisch Thanks for `attractor`.
I'm using it and need to export results as CSV, so I thought it would be useful for other users too.

### CODE REVIEW

File: lib/attractor/value.rb [hunk 1]
```diff
@@ -19,6 +19,10 @@ def current_commit
       history&.first&.first
     end
 
+    def score
```
Comment (julianrubisch) [inline lib/attractor/value.rb:22]: I totally see the intent here and actually think this is a nice improvement, but I'm thinking that the name itself is maybe a bit too general.

In the future, there might be more "scores" that then might need disambiguation - but in fairness, I can't come up with something better at this moment...

**Review by julianrubisch — COMMENTED** [review]
Thanks again for this awesome contribution!

I need to actually check it out and run it, but I haven't spotted any problem.

I only had one minor naming comment.

**Comment by julianrubisch** [comment]
Oh, and please run `bin/standardize`!

**Comment by jodosha** [comment]
@julianrubisch Thanks for the review.

I ran `bin/standardize`. It fixed some code, but also crashed with this error:

```
⚡ bin/standardize
(node:57148) [DEP0128] DeprecationWarning: Invalid 'main' field in '/Users/jodosha/.npm/_npx/3b9ec74976921f5f/node_modules/find-parent-dir/package.json' of 'find-parent-dir.js'. Please either fix that or report it to the module author
(Use `node --trace-deprecation ...` to show where the warning was created)
Error: Failed to load plugin 'prettier' declared in '../../.eslintrc.json': Cannot find module 'eslint-plugin-prettier'
Require stack:
- /Users/jodosha/.npm/_npx/3b9ec74976921f5f/node_modules/prettier-standard/src/vendor/__placeholder__.js
Referenced from: /Users/jodosha/Code/attractor/.eslintrc.json
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.resolve (node:internal/modules/cjs/helpers:108:19)
    at Object.resolve (/Users/jodosha/.npm/_npx/3b9ec74976921f5f/node_modules/eslint/lib/shared/relative-module-resolver.js:44:50)
    at ConfigArrayFactory._loadPlugin (/Users/jodosha/.npm/_npx/3b9ec74976921f5f/node_modules/eslint/lib/cli-engine/config-array-factory.js:959:39)
    at ConfigArrayFactory._loadExtendedPluginConfig (/Users/jodosha/.npm/_npx/3b9ec74976921f5f/node_modules/eslint/lib/cli-engine/config-array-factory.js:779:29)
    at ConfigArrayFactory._loadExtends (/Users/jodosha/.npm/_npx/3b9ec74976921f5f/node_modules/eslint/lib/cli-engine/config-array-factory.js:725:29)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (/Users/jodosha/.npm/_npx/3b9ec74976921f5f/node_modules/eslint/lib/cli-engine/config-array-factory.js:660:25)
    at _normalizeObjectConfigDataBody.next (<anonymous>)
    at ConfigArrayFactory._normalizeObjectConfigData (/Users/jodosha/.npm/_npx/3b9ec74976921f5f/node_modules/eslint/lib/cli-engine/config-array-factory.js:596:20)
    at _normalizeObjectConfigData.next (<anonymous>) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/jodosha/.npm/_npx/3b9ec74976921f5f/node_modules/prettier-standard/src/vendor/__placeholder__.js'
  ],
  messageTemplate: 'plugin-missing',
  messageData: {
    pluginName: 'eslint-plugin-prettier',
    resolvePluginsRelativeTo: '/Users/jodosha/.npm/_npx/3b9ec74976921f5f/node_modules/prettier-standard/src/vendor',
    importerName: '../../.eslintrc.json'
  }
}
```

File: lib/attractor/value.rb (hunk 1 above)
Comment (jodosha) [reply lib/attractor/value.rb:22]: @julianrubisch Do you want me to revert this?

Comment (julianrubisch) [reply lib/attractor/value.rb:22]: No! I'm just thinking about a better name... give me a few days, I'm kinda swamped here :-)

**Comment by julianrubisch** [comment]
Oh, sorry, I'll take a look at whats wrong here!

**Comment by julianrubisch** [comment]
Merged. Sorry for the slow turnaround here!

**Pull request merged**